### PR TITLE
ci(dependabot): add devDependencies auto-updates

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We would like to auto-update devDependencies, with the due exceptions (like `typescript` and `lint`).

### CODE REVIEW

**Comment by AntonioVentilii-DFINITY** [comment]
@peterpeterparker please have a look if there are packages to be included/excluded among the exceptions

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,4 +6,20 @@ updates:
...
+    schedule:
+      interval: "weekly"
+    target: "dev"
+    ignore:
+      - dependency-name: "@dfinity/*"
+      - dependency-name: "@types/*"
+      - dependency-name: "@typescript-eslint/*"
+      - dependency-name: "eslint"
+      - dependency-name: "eslint-*"
+      - dependency-name: "sass"
```
Comment (peterpeterparker) [inline .github/dependabot.yml:22]: I think we can try to not ignore sass, svelte, and vite. WDYT?

**Review by peterpeterparker — APPROVED** [review]
Coolio, thanks. Already approving, just a question.

File: .github/dependabot.yml (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply .github/dependabot.yml]: very brave eheh! agreed and done

**Pull request merged**